app/models/customer.py

```diff
@@ -0,0 +1,21 @@
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    __tablename__ = "customers"
+    customer_id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.Integer)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, default=datetime.utcnow())
+    videos_checked_out_count = db.Column(db.Integer, default=0)
```

Comment:
Instead of this being a column you can use the rentals to count that.  

To do that it's helpful to add a relationship:
```suggestion
    rentals = db.relationship('Rental', back_populates='customer', lazy=True)
```